scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/vfs/Database.scala

```diff
@@ -19,17 +18,7 @@ object Database {
 @data
 class Database(entries: List[Entry]) {
   def toSchema: s.Database = {
-    val sentries = entries.map(ventry => s.Attributes.parseFrom(ventry.bytes))
+    val sentries = entries.flatMap(ventry => s.Database.parseFrom(ventry.inputStream).entries)
```

Comment:
Am I right in thinking that the change from `.bytes` to `.inputStream` here is done for optimization purposes?